src/main/scala/intellij/haskell/action/DownloadCabalConfigFile.scala

```diff
@@ -0,0 +1,17 @@
+package intellij.haskell.action
+
+import com.intellij.openapi.actionSystem.{AnAction, AnActionEvent}
+import intellij.haskell.external.component.HaskellComponentsManager.refreshGlobalProjectInfo
+import intellij.haskell.external.component.PathComponent
+
```

Comment:
>Ok, so maybe download file here in GlobalProjectInfo is cool enough? Cause the global project info will always be loaded when a project is created or opened right?

Yes, that's okay.

Well, then it's better to create or reuse existing action which refreshes everything. We could use the action which restarts the repls for this. We have many dependencies. Also user  can change dependending libraries or Stack LTS. We also have action for downloading sources. I will think about it...